ShelterModule/Services/Implementations/Shelters/ShelterCommand.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Database;
+using Microsoft.EntityFrameworkCore;
+using ShelterModule.Models.Shelters;
+using ShelterModule.Services.Interfaces.Shelters;
+
+namespace ShelterModule.Services.Implementations.Shelters;
+
+public sealed class ShelterCommand : IShelterCommand
+{
+    private readonly PetShareDbContext _context;
+
+    public ShelterCommand(PetShareDbContext context)
+    {
+        _context = context;
+    }
+
+    public async Task AddAsync(Shelter shelter)
+    {
+        _context.Add(shelter.ToEntity());
+        await _context.SaveChangesAsync();
+    }
+
+    public async Task RemoveAsync(Shelter typeObject)
+    {
+        _context.Remove(typeObject);
```

Comment:
That won't work. We need to get the entity that we want to delete from database, and then call `_context.Remove()`.